applications/asset_tracker_v2/src/events/debug_module_event.h

```diff
@@ -34,12 +34,12 @@ enum debug_module_event_type {
 	 *  MODEM_EVT_INITIALIZED which is not sent due to the modem module being disabled for
 	 *  QEMU x86 builds.
 	 */
-	DEBUG_EVT_QEMU_X86_INITIALIZED,
```

Comment:
The event description needs an update to reflect the event renaming.

Reply:
done